README.md

```diff
@@ -33,6 +33,46 @@ Check env availbility by the following command:
 rally env check
 ```
 
+## Scenarios
```

Comment:
Don't do this. It is hard to maintain, easy to miss something. As for plugins reference, we will have a separate page here - https://xrally.org/plugins/kubernetes/overview/ (like, openstack and docker have) as soon as we get first release (let's finish namespace staff and do it). Such pages are auto-generated, so as soon as new plugin appear, it will be added there, as soon as the docsting or arguments are changed, the proper section will be updated at web-page.

To be more user-friendly, use can describe in README all_in_one task which will contain a set of workloads which tests all basics of a k8s cluster.

Reply:
Also, you can mention `rally plugin list --platform kubernetes` command